pkgs/top-level/octave-packages.nix

```diff
@@ -0,0 +1,91 @@
+# This file contains the GNU Octave add-on packages set.
+# Each attribute is an Octave library.
+# Expressions for the Octave libraries are supposed to be in `pkgs/development/octave-modules/<name>/default.nix`.
+
+{ pkgs
+, lib
+, stdenv
+, fetchurl
+, newScope
+, octave
+, lapack, blas
+, gfortran
+# Use the same python that octave is using
+, python
+# Use the same Java that octave is using
+, enableJava
+, jdk
+, gnuplot
+, texinfo
+, nettle
+}:
+
+let
+
+  isOctaveFull = octave.enableQt;
```

Comment:
You can even inherit these attributes in the `newScope` and then use them in the derivations without changing the names !

Reply:
See https://github.com/NixOS/nixpkgs/pull/108562/commits/a9597730631bb5f8b0086a273811a930cbb7b6a0